### PR TITLE
Add specifications about the FO payment of a newly created order

### PR DESCRIPTION
**Pull request by LouiseBonnard**
Add missing specifications related to this bug, https://github.com/PrestaShop/PrestaShop/issues/20977.

How should the user behave when he decides to create a new order for a customer and to process its payment via the checkout in the front office?

### CODE REVIEW

**Comment by LouiseBonnard** [comment]
@MatShir, do you think we should prioritize it for 1.7.8?

**Review by marionf — APPROVED** [review]

**Review by Robin-Fischer-PS — COMMENTED** [review]
Small feedback/question : 
If I understand well, as long as the order is not paid through checkout in FO, the order is not displayed in BO ? That might be a problem, if the merchant wants to update the order before the customer pays it.

**Comment by LouiseBonnard** [comment]
Hi @Robin-Fischer-PS, thanks for the review. I understand what you mean, I will improve my specifications to clear things up. Usually, when you're creating a new order in the back office, you go through the whole page and click 'Create the order'.

What I meant here was that, if the user directly processes the payment in the front office with this link (without clicking 'Create the order' in the back office), then the order is automatically created in the back office and he no longer has to click on 'Create the order'.

File: back-office/orders/orders/Order Add/Add new Order.md [hunk 1]
```diff
@@ -206,22 +206,30 @@ A soft "deleted" address can be used in the cart / order.
...
+
 ### Send an email to the customer with the link to process the payment
 
 This button allows to send an email to the customer with the link to process the payment
 It's not related to the payment method selected.
-The link opens the front-office with the cart corresponding to the order created.
+
+Next to this button is a hyperlink to allow the user to proceed to the payment directly in the front office. Clicking this hyperlink opens a new tab that redirects the user to the payment step of the checkout process for this order - the user does not need to fill all the personal information, addresses, or shipping method but he must be able to navigate through those previous steps before going back to the payment.
+
+Processing the payment of this order through the checkout in the front office immediately creates the order in the back office. As a consequence, the user is not creating a new order anymore and must go in the Orders > Orders section of his/her back office to reach this order.
```
Comment (LouiseBonnard) [inline back-office/orders/orders/Order Add/Add new Order.md:225]:
```suggestion
Processing the payment through the checkout in the front office automatically creates the order and adds it to the orders listing in the back office. As a consequence, the merchant does not have to complete the ongoing creation of the new order.

If the payment is not processed right away, the merchant must keep on completing the creation form in the back office and click 'Create new order' to add it to the orders listing.
```

Comment (LouiseBonnard) [reply back-office/orders/orders/Order Add/Add new Order.md:225]: @Robin-Fischer-PS

Comment (Robin-Fischer-PS) [reply back-office/orders/orders/Order Add/Add new Order.md:225]: Thanks for the edit, it clears things up !

**Review by Robin-Fischer-PS — APPROVED** [review]

**Comment by marionf** [comment]
@LouiseBonnard Could you resolve the conflicts please ?

**Comment by LouiseBonnard** [comment]
@marionf, I've just used GitHub's conflict editor, this should be okay now. I let you check this PR one last time and merge it, thank you very much!

**Comment by marionf** [comment]
Thank you @LouiseBonnard :tada:

**Pull request merged**